tests/utils/test_cache_utils.py

```diff
@@ -552,14 +437,19 @@ def test_cache_copy(self):
         for prompt in prompts:
             new_inputs = tokenizer(INITIAL_PROMPT + prompt, return_tensors="pt").to("cuda")
             past_key_values = copy.deepcopy(prompt_cache)
-            outputs = model.generate(**new_inputs, past_key_values=past_key_values, max_new_tokens=40)
+            outputs = model.generate(
+                **new_inputs, past_key_values=past_key_values, max_new_tokens=40, disable_compile=True
+            )
             response = tokenizer.batch_decode(outputs)[0]
             responses.append(response)
 
         EXPECTED_DECODED_TEXT = [
-            "You are a helpful assistant. Help me to write a blogpost about travelling.\n\nTraveling is an enriching experience that broadens our horizons and exposes us to new cultures, landscapes, and people. Whether it's a week",
```

Comment:
if we checkout to the commit that added this test, we get a different output 👀 possibly due to different hardware/software?